data/js/tomahawk.js

```diff
@@ -49,9 +51,9 @@ RSVP.on('error', function (reason) {
         resolverName = Tomahawk.resolver.instance.settings.name + " - ";
     }
     if (reason) {
-        console.error(resolverName + 'Uncaught error:', reason);
+        Tomahawk.error(resolverName + 'Uncaught error:', reason);
     } else {
-        console.error(resolverName + 'Uncaught error: error thrown from RSVP but it was empty');
+        Tomahawk.error(resolverName + 'Uncaught error: error thrown from RSVP but it was empty');
```

Comment:
Does this keep line numbers in tact? I know they are screwed on desktop anyway, but might get interesting should we ever switch to QWebEngine or maybe you get more meaningful errors in Android already..